docs/installation.rst

```diff
@@ -0,0 +1,69 @@
+Installation
+============
+
+Graphene-Django takes a few seconds to install and set up.
+
+Requirements
+------------
+
+Graphene-Django currently supports the following versions of Django:
+
+* Django 2.0 >
+
+Installation
```

Comment:
General comment on the newly added material here - it duplicates some of the work in `tutorial-plain`. Is there a plan to harmonize the both together. If not, happy to volunteer for that.

Reply:
The tutorials are quite big, I want to generalize things on the other pages, then you or I can go and rewrite the tutorials to conform with it better.